kura/distrib/src/main/ant/build_equinox_distrib.xml

```diff
@@ -401,25 +401,25 @@ fi]]>
                         prefix="${build.output.name}/${user.config.folder}/" />
 
 			<zipfileset file="${project.build.directory}/${build.output.name}/kura_install.sh"
-                        prefix="${build.output.name}/${install.folder}/" />
+                        prefix="${build.output.name}/${install.folder}/" filemode="777" />
 
```

Comment:
Maybe 755 or 700 is more appropriate. Could we move this chmod operation to the kura_install.sh?

Reply:
Good point. So, leave it in `build_equinox_distrib.xml`, but please change the mode. :-)